setup.py

```diff
@@ -8,10 +8,10 @@
     packages = ['ticketgraph'],
     package_data = { 'ticketgraph' : [ 'htdocs/*.*', 'templates/*.*' ] },
 
-    author = 'Colin Snover',
-    author_email = 'tracplugins@zetafleet.com',
+    author = 'Fabrizio Parrella',
```

Comment:
Sorry, I initially meant to create a new plugin all together that is why I did change it.  reverting that change